samples/dashboards/component-options/caption/demo.css

```diff
@@ -1,3 +1,6 @@
+@import url(“https://code.highcharts.com/css/dashboards/gui.css”);
```

Comment:
There is something wrong with the way it is imported I think the `“` is wrong.
Please make it as you did in the `component-options/events/demo.css`

Reply:
Done